feeds/capcotentin.fr.dmfr.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://dmfr.transit.land/json-schema/dmfr.schema-v0.5.0.json",
+  "feeds": [
+    {
+      "id": "f-cap~cotentin~transit",
+      "spec": "gtfs",
+      "urls": {
+        "static_current": "https://www.data.gouv.fr/fr/datasets/r/2e97c9b3-a59f-42dd-9b9e-a232fa771f21"
+      },
+      "license": {
+        "url": "https://www.etalab.gouv.fr/licence-ouverte-open-licence/"
+      }
+    },
+    {
+      "id": "f-cap~cotentin~transit~rt",
+      "spec": "gtfs-rt",
+      "urls": {
+        "realtime_vehicle_positions": "https://pysae.com/api/v2/groups/transdev-cotentin/gtfs-rt",
+        "realtime_trip_updates": "https://pysae.com/api/v2/groups/transdev-cotentin/gtfs-rt",
+        "realtime_alerts": "https://pysae.com/api/v2/groups/transdev-cotentin/gtfs-rt"
+      },
+      "license": {
+        "url": "https://www.etalab.gouv.fr/licence-ouverte-open-licence/"
+      }
+    }
+  ],
+  "operators": [
+    {
+      "onestop_id": "o-cap~cotentin~transit",
+      "name": "Cap Cotentin",
+      "website": "https://www.capcotentin.fr",
+      "associated_feeds": [
```

Comment:
Also associate this operator with `f-sp9x-normandie`

It's fine for an operator to group together multiple static feeds that can only be used exclusively.